circle.yml

```diff
@@ -20,7 +20,7 @@ test:
 
 deployment:
   registry:
-    branch: master
+    branch: channel/eslint-2
```

Comment:
(And of course now I see something to note :))

I don't think we want to have to edit the circle.yml in all the deployment branches. I think this value can be something like `master|channel/*` to trigger deployment for all the branches we want it to.